foyer/tests/test_forcefield.py

```diff
@@ -287,3 +287,18 @@ def test_write_xml(filename):
 
     assert sigma_factor_pre == sigma_factor_post
     assert epsilon_factor_pre == epsilon_factor_post
+
+@pytest.mark.parametrize("filename", ['ethane.mol2', 'benzene.mol2'])
+def test_write_xml_multiple_periodictorsions(filename):
+    cmpd = pmd.load_file(get_fn(filename), structure=True)
+    ff = Forcefield(forcefield_files=get_fn('oplsaa_multiperiodicitytorsion.xml'))
+    typed_struc = ff.apply(cmpd, assert_dihedral_params=False)
+    typed_struc.write_foyer(filename='multi-periodictorsions.xml', forcefield=ff, unique=True)
+
+    partial_ff = Forcefield(forcefield_files='multi-periodictorsions.xml')
+    typed_by_partial = partial_ff.apply(cmpd, assert_dihedral_params=False)
+
+    assert len(typed_struc.bonds) == len(typed_by_partial.bonds)
+    assert len(typed_struc.angles) == len(typed_by_partial.angles)
+    assert len(typed_struc.dihedrals) == len(typed_by_partial.dihedrals)
+
```

Comment:
Can we also so some checking to make sure that the variables like `periodicity2`/`phase2`/`k2` variables still exist in the force field object after it's read back in (and ideally are of similar value as well)?